.github/workflows/test.yml

```diff
@@ -84,12 +84,13 @@ jobs:
       - name: cmake
         uses: jwlawson/actions-setup-cmake@v2
         with:
-          cmake-version: '3.31.x'
+          cmake-version: "3.31.x"
 
-      - name: install build dependencies (macos-latest, wasmer_wamr)
-        if: matrix.os == 'macos-latest' && matrix.target == 'wasmer_wamr'
-        shell: bash
-        run: brew install cmake ninja
+      - name: install build dependencies (wasmer_wamr)
+        if: matrix.target == 'wasmer_wamr'
+        run: |
+          brew install llvm
```

Comment:
cmake is installed above already, ninja's included in it, and llvm is needed for `llvm-objcopy` that wasmer depends on.